### PR TITLE
shadow for red planet element

### PR DESCRIPTION
**Pull request by poornima005**
## Description
Please include a summary of the changes and the purpose of this PR.

- What does this PR do?
- Why is it needed?

## Screenshots
Include screenshots for **both desktop and mobile** versions.

### Desktop:
[Add screenshot here]

### Mobile:
[Add screenshot here]

## Approach
Briefly explain the approach taken to implement the changes. 

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/game_frontend/src/pages/Dashboard/Airdrop/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Airdrop = () => {
             />
           </div>
           <p>token is on the market</p>
-          <img src={astrouant} alt="astrouant" className="astrouant-img" />
+          <img src="../.../../../../../public/assets/images/astronaut.webp" alt="astrouant" className="astrouant-img" />
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/Airdrop/index.tsx:23]: use path `/assets/images/astronaut.webp`

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

File: src/StarDustAdventures_frontend/src/components/landing/gameConcept/GameConcept.jsx [hunk 2]
```diff
@@ -26,13 +26,14 @@ const GameConcept = () => {
...
-        <div className="gcs-sub-center" >
-            <img src="red_planet.png" alt="red planet" className="gcs-sc-img" />
-            <div className="gcs-sc-text-item" data-aos="fade-up" data-aos-duration={1500}>
-                <h4 className="gcs-sc-title">{gameConceptText.center.title}</h4>
-                <p className="gcs-sc-text">{gameConceptText.center.text}</p>
-            </div>
-        </div>
+        <div className="gcs-sub-center">
+    <img src="red_planet.png" alt="red planet" className="gcs-sc-img" />
+    <div className="gcs-sc-text-item" data-aos="fade-up" data-aos-duration={1500}>
```
Comment (shadabkhan16) [inline src/StarDustAdventures_frontend/src/components/landing/gameConcept/GameConcept.jsx:31]: Proper indentation

File: src/StarDustAdventures_frontend/src/components/landing/gameConcept/gameConcept.css [hunk 3]
```diff
@@ -45,4 +45,8 @@
 }
 .gcs-sc-title{
     @apply mb-3 xl:text-3xl md:text-xl font-semibold text-center
-}
\ No newline at end of file
+}
+.gcs-sc-img {
+    box-shadow: 0 0 50px 35px rgba(255, 0, 0, 0.21); /* Uniform, light red shadow */
```
Comment (shadabkhan16) [inline src/StarDustAdventures_frontend/src/components/landing/gameConcept/gameConcept.css:50]: Try this tailwind CSS utility classes
```
shadow-[0_0_50px_35px_rgba(255,0,0,0.21)] rounded-full
```

File: src/game_frontend/src/components/ui/Progressbar/index.css [hunk 4]
```diff
@@ -37,10 +37,30 @@
...
+    .progress-bar {
+        @apply w-3 h-[22rem] relative overflow-hidden sm:rotate-0 rotate-90 mr-[7rem] mt-[60rem];
+    }
+   
+    }
+    
+@media (max-width:640px) {
+    .progress-bar {
+        @apply w-3 h-[22rem] relative overflow-hidden sm:rotate-0 rotate-90 mr-[12rem] mt-[60rem];
     }
```
Comment (shadabkhan16) [inline src/game_frontend/src/components/ui/Progressbar/index.css:64]: Try this 
```
w-[26rem] h-[2rem] relative overflow-hidden sm:rotate-0 rotate-90 mr-[7rem] mt-[60rem] 
  md:w-[26rem] md:h-[2rem] 
  lg:w-3 lg:h-[22rem] lg:mr-[12rem] 
   xl:mr-[7rem]
```

**Pull request closed without merging**